docs/cli/overview.md

```diff
@@ -0,0 +1,101 @@
+
+The Aspire CLI (`aspire` command) is a cross-platform tool that provides command-line functionality to create, manage, run, and publish polyglot Aspire projects. Use the Aspire CLI to streamline development workflows and coordinate services for distributed applications.
+
+The Aspire CLI is an interactive-first experience.
+
+- [Install .NET Aspire CLI](install.md)
+
+## Commands
+
+The following table describes the basic commands provided by Aspire:
+
+| Command                                                    | Status  | Function                                                                  |
+|------------------------------------------------------------|---------|---------------------------------------------------------------------------|
+| [`aspire add`](#add-integrations)                          | Stable  | Add an integration to the Aspire project.                                 |
+| [`aspire new`](#create-projects)                           | Stable  | Create an Aspire sample project from a template.                          |
+| [`aspire run`](#run-the-apphost-project)                   | Stable  | Run an Aspire apphost in development mode.                                |
+| [`aspire exec`](#run-commands-in-resource-context-preview) | Preview | Similar to the `aspire run` command, but passes commands to the apphost.  |
+| [`aspire deploy`](#deploy-aspire-applications-preview)     | Preview | Deploys the artifacts created by `aspire publish`.                        |
+| [`aspire publish`](#publish-aspire-applications-preview)   | Preview | Generates deployment artifacts for an Aspire apphost project.             |
+
+<!-- These commands aren't used yet
+
+| [`aspire config`](#configure-aspire-environment)         | Stable  | Configures the Aspire environment.                                        |
+| `aspire init`                                            | Future  | ... |
+
+-->
+
+## Create projects
+
+The `aspire new` command is an interactive-first CLI experience, and is used to create one or more Aspire projects. As part of creating a project, Azure CLI ensures that the latest Aspire project templates are installed into the `dotnet` system.
+
+<!-- Add asciinema here -->
+
+Use the `aspire new` command to create an Aspire project from a list of templates. Once a template is selected, the name of the project is set, and the output folder is chosen, `aspire` downloads the latest templates and generates one or more projects.
+
+While command line parameters can be used to automate the creation of an Aspire project, the Aspire CLI is an interactive-first experience.
+
+## Run the apphost project
+
+The `aspire run` command runs the apphost project in development mode, which configures the Aspire environment, builds the apphost, launches the web dashboard, and displays a terminal-based dashboard. This is similar to running the apphost project in your IDE of choice, however, a terminal dashboard is also visible.
```

Comment:
I know Maddy and Fowler, really like "AppHost", but I'm still not convinced it's correct. It's an overload phrase, are we talking about the concept of the orchestrator, the project itself, the project template, the AppHost.cs file? There are way too many things that this could be referring to, and it's becoming ambiguous. I've historically tried to combat that by saying:

- AppHost project: when talking about the project (modified template).
- AppHost template: when talking about the template (starting point).
- AppHost.cs: when talking about the entry point.
- App host: when talking about it conceptually.

This isn't perfect, but one thing we haven't done is call it `apphost`.